linux/open_vm_tools/check_inbox_drivers_unchanged.yml

```diff
@@ -5,44 +5,20 @@
 # open-vm-tools from source. Inbox drivers to be checked include vmxnet3,
 # vmw_pvscsi, vmw_balloon.
 #
-- name: "Set fact of guest OS inbox drivers directory path"
-  ansible.builtin.set_fact:
-    guest_inbox_drivers_dir: "/lib/modules/{{ guest_os_ansible_kernel }}/kernel/drivers"
-
-- name: "Get guest OS inbox drivers' filenames"
-  ansible.builtin.shell: "modinfo -F filename {{ item }}"
-  ignore_errors: true
+- name: "Get guest OS inbox drivers' files after installing open-vm-tools from source"
+  ansible.builtin.shell: "modinfo -F filename vmxnet3 vmw_pvscsi vmw_balloon"
```

Comment:
Will this command fail when getting any of these 3 specified driver fails? If not, then only check output lines seems not secure for getting expected result. Or if there is any chance that output lines are 3 while the results are not expected actually.

Reply:
If it failed to get file of any of the 3 drivers, the command will fail, and it will jump to test rescue. These 3 drivers must exist in guest OS, otherwise it would be an OS issue. As long as they exist, this command will return their files in order of "vmxnet3 vmw_pvscsi vmw_balloon". 

Here we only check these 3 drivers files are same as those before installing open-vm-tools, that is what we expect.